services/bedrockagentruntime/src/main/resources/codegen-resources/customization.config

```diff
@@ -1,4 +1,76 @@
 {
     "enableGenerateCompiledEndpointRules": true,
-    "enableFastUnmarshaller": true
-}
+    "enableFastUnmarshaller": true,
+    "useLegacyEventGenerationScheme": {
+        "FlowResponseStream": {
```

Comment:
Just to check my understanding, what happens if/when Bedrock adds a new event to this eventstream? Will it default to `DISABLED` or since the mapping doesn't exist it will fail?

Reply:
New event members added to existing eventstreams will just have their shape's renamed to ensure uniqueness, there aren't any issues mixing the legacy behavior with the new behavior in the same eventstream.